drake/doc/ubuntu.rst

```diff
@@ -30,13 +30,18 @@ Install the prerequisites::
     # Ubuntu 14.04 LTS (Trusty)
     sudo apt-get install cmake-curses-gui
 
+    # These are not required for the basic "make" rule.  We suggest that you
+    # wait to do this step until you actually need these tools.  (We intend
+    # to switch to non-pip instructions once that approach has been tested.)
     sudo pip install -U cpplint Sphinx
 
 Download the external dependencies::
 
     cd drake-distro
-    make options  # use the GUI to choose which externals you want, then press 'c' to configure, then 'g' to generate makefiles and exit
-    make download-all
+    env CXX=g++-4.9 CC=gcc-4.9 PATH=/usr/lib/ccache:$PATH make options
```

Comment:
However, I don't understand the reason for including ccache in these install steps.  It's not required, and presumably anyone who wants to use it has already PATHed it in their dotfiles.